packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -20,25 +20,47 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * Indicates if the DetailList should be a description list for label-value pairs.
+   */
+  isDescriptionList: PropTypes.bool,
 };
 
 const defaultProps = {
   title: undefined,
   children: undefined,
+  isDescriptionList: false,
```

Comment:
I'm still thinking about the naming of this key-value pair list...

Reply:
I'm hesitant to use the implementation detail in the name of the prop; we could swap out the implementation later (if we need to).

How about if we rename this `isDescriptionList` prop to `type` with enum values: `single`, `name-value-pair`. Still open to the enum values suggestion. Ideally, it should describe the options of either single value or pairs.

From here, we can determine the logic to use <ul> vs <dl>.

Open to feedback or other suggestions too.